Tools/Scripts/hooks/prepare-commit-msg

```diff
@@ -3,13 +3,39 @@
 import os
 import subprocess
 import sys
+from webkitpy.common.checkout.diff_parser import DiffParser
```

Comment:
This isn't going to work reliably.

You'll need to make sure `SCRIPTS` is on the Python path before importing because the user might actually be setting up a different repository with WebKit's tooling. That's why the old line 18/new line 44 invokes `prepare-ChangeLog` from `SCRIPTS` specifically.